### PR TITLE
Update locale-formula to 0.4.0

### PR DESCRIPTION
**Pull request by deneb-alpha**
- Update main README.md with new links to salt official documentation and rebranding for SUSE Multi-Linux Manager
- Update locale-formula to 0.4.0
  * Add spec and changes in git
  * Update version in Makefile
  * Update link in README.rst
  * Run spec-cleaner and prevent rejects from factory-auto bot

### CODE REVIEW

File: locale-formula/locale-formula.spec [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+
+%define fname locale
+Name:           %{fname}-formula
+Version:        0.4.0
+Release:        0
+Summary:        Locale Salt Formula for SUSE Manager
```
Comment (nodeg) [inline locale-formula/locale-formula.spec:23]:
```suggestion
Summary:        Locale Salt Formula for SUSE Multi-Linux Manager
```

File: locale-formula/locale-formula.spec [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+License:        GPL-3.0-only
+Group:          System/Management
+URL:            https://github.com/SUSE/salt-formulas
+Source0:        %{name}-%{version}.tar.gz
+Requires:       salt-master
+BuildArch:      noarch
+
+%description
+Salt Formula for SUSE Manager. Sets up the locale.
```
Comment (nodeg) [inline locale-formula/locale-formula.spec:32]:
```suggestion
Salt Formula for SUSE Multi-Linux Manager. Sets up the locale.
```

**Review by mcalmer — APPROVED** [review]

**Review by nodeg — APPROVED** [review]
Fine with me. The only issue I see is the commit message beginning with `-` and it being too long 😉 other than that LGTM.

**Comment by deneb-alpha** [comment]
> Fine with me. The only issue I see is the commit message beginning with `-` and it being too long 😉 other than that LGTM.

How I should amend the commit message?
now I used this:
~~~
- Update main README.md with new links to salt official documentation and
  rebranding for SUSE Multi-Linux Manager
- Update locale-formula to 0.4.0
  * Add spec and changes in git
  * Update version in Makefile
  * Update link in README.rst
  * Run spec-cleaner and prevent rejects from factory-auto bot
~~~

**Pull request merged**